backend/scripts/k8s_vector_cleanup.py

```
@@ -0,0 +1,228 @@
+            cleanup_old_chat_collections,
+        )
+        from open_webui.models.knowledge import Knowledges
+
+        logger.info("✓ Successfully imported cleanup modules")
+
+        # Test database connectivity
+        logger.info("🔌 Testing database connectivity...")
+        try:
+            # Try to get knowledge bases as a connectivity test
+            knowledge_bases = Knowledges.get_knowledge_bases()
+            logger.info(f"✓ Database connection successful")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)